production/helm/loki/values.yaml

```diff
@@ -337,7 +337,7 @@ loki:
   memberlistConfig: {}
   # -- Extra memberlist configuration
   extraMemberlistConfig: {}
-  # -- Tenants list to be created on nginx htpasswd file, with name and password keys
+  # -- Tenants list to be created on nginx htpasswd file, with name and password or passwordHash keys
```

Comment:
Could you add a example values in the comments here? I tried this feature locally and I had to check the code first how that working.